core/src/main/scala/org/apache/spark/deploy/history/HistoryServer.scala

```diff
@@ -59,7 +59,7 @@ class HistoryServer(
   private val retainedApplications = conf.get(History.RETAINED_APPLICATIONS)
 
   // How many applications the summary ui displays
-  private[history] val maxApplications = conf.get(HISTORY_UI_MAX_APPS);
+  private[history] val maxApplications = conf.get(HISTORY_UI_MAX_APPS)
```

Comment:
Does this show any warnings? If not, let's don't do this. It will make difficult to maintain e.g.) backporting or revert but the gain is just cosmetic stuffs

Reply:
OK ~ got it ~